src/main/java/com/booking/superride/entity/CustomerDetails.java

```diff
@@ -0,0 +1,19 @@
+package com.booking.superride.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "customer_details")
+public class CustomerDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "customer_details_customerid_seq")
+    @SequenceGenerator(name = "customer_details_customerid_seq", sequenceName = "customer_details_customerid_seq", allocationSize = 1)

```

Comment:
With new versions of hibernate , no need to declare `@SequenceGenerator` , also not necessary to mention generation in line 12